src/contract_wrappers/modules/transfer_manager/count_transfer_manager_wrapper/common.ts

```diff
@@ -0,0 +1,143 @@
+import {
+  CountTransferManagerContract_3_0_0,
+  Web3Wrapper,
+  BigNumber,
+  PolyResponse,
+} from '@polymathnetwork/abi-wrappers';
+import { numberToBigNumber, parseTransferResult, valueToWei } from '../../../../utils/convert';
+import ContractFactory from '../../../../factories/contractFactory';
+import { ErrorCode, Perm, TransferResult, TxParams } from '../../../../types';
+import ModuleWrapper from '../../module_wrapper';
+import assert from '../../../../utils/assert';
+
+export namespace CountTransferManagerTransactionParams {
+  export interface ChangeHolderCount extends ChangeHolderCountParams {}
+}
+
```

Comment:
As there isn't a 3.1.0 CountTM, you could move all the events to this common class.